index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
- [ ] Your pull request description/summary should most timesbe in bullet points for ease of read.